Content.Server/Shuttles/Systems/RadarConsoleSystem.cs

```diff
@@ -30,6 +32,13 @@ protected override void UpdateState(EntityUid uid, RadarConsoleComponent compone
         EntityCoordinates? coordinates = onGrid ? xform.Coordinates : null;
         Angle? angle = onGrid ? xform.LocalRotation : null;
 
+        //handheld test
+        if (HasComp<PowerCellDrawComponent>(uid))
+        {
+            coordinates = new EntityCoordinates(uid, Vector2.Zero);
+            angle = Angle.Zero + MathHelper.DegreesToRadians(180);
+        }
+
         // Use ourself I guess.
         if (TryComp<IntrinsicUIComponent>(uid, out var intrinsic))
         {
```

Comment:
1. Did you ask the frontier coder as a courtesy.
2. This shouldn't be using a separate UI path to intrinsic UIs considering they're doing the same thing.